derp/derphttp/derphttp_client.go

```diff
@@ -185,6 +185,18 @@ func (c *Client) useHTTPS() bool {
 	if c.url != nil && c.url.Scheme == "http" {
 		return false
 	}
+	// Turn on development mode and connect to DERP using plain HTTP.
+	// Questions:
+	// * Should this be a command line option or an envknob?
+	// * If envknob, should it be stubbed out for iOS, etc.?
+	// * Should it be a port number instead of a boolean?
```

Comment:
My preference would be to have a simpler setting so that there's less chances for a developer to get into a broken state, but I haven't done enough DERP-related development to have an opinion as to whether more flexibility is needed (would you want to run multiple derpers locally?).

Reply:
Ooh, good point. With the mesh code I can see wanting to run multiple derps locally for local development. But let's leave that for when we need it.